### ISSUE TITLE
Support Node 0.10+

### ISSUE THREAD

**Issue by vladikoff** [Rantanen/node-jitterbuffer#4]
Hey, great module!
Getting this error:
```
npm WARN engine protobuf@0.8.7: wanted: {"node":">= 0.8.0 < 0.9.0"} (current: {"node":"v0.10.26","npm":"1.4.3"})
npm ERR! error rolling back Error: ENOTEMPTY, rmdir '/Users/vladikoff/tmp/mumble-web/node_modules/mumble'
npm ERR! error rolling back  mumble@0.0.2 { [Error: ENOTEMPTY, rmdir '/Users/vladikoff/tmp/mumble-web/node_modules/mumble']
npm ERR! error rolling back   errno: 53,
npm ERR! error rolling back   code: 'ENOTEMPTY',
npm ERR! error rolling back   path: '/Users/vladikoff/tmp/mumble-web/node_modules/mumble' }
npm ERR! notsup Unsupported
npm ERR! notsup Not compatible with your operating system or architecture: jitterbuffer@0.1.3
npm ERR! notsup Valid OS:    linux
npm ERR! notsup Valid Arch:  x64
npm ERR! notsup Actual OS:   darwin
npm ERR! notsup Actual Arch: x64

npm ERR! System Darwin 13.1.0
npm ERR! command "node" "/usr/local/bin/npm" "install" "mumble"
npm ERR! cwd /Users/vladikoff/tmp/mumble-web
npm ERR! node -v v0.10.26
npm ERR! npm -v 1.4.3
npm ERR! code EBADPLATFORM
```

**Comment by Rantanen** [comment]
Fixed by @Prior99 73dc3d3

**Comment by Rantanen** [comment]
Actually I take that back!

The issue reported in the title was fixed by Prior99. However...

    npm ERR! notsup Not compatible with your operating system or architecture: jitterbuffer@0.1.3
    npm ERR! notsup Valid OS:    linux
    npm ERR! notsup Valid Arch:  x64
    npm ERR! notsup Actual OS:   darwin
    npm ERR! notsup Actual Arch: x64

The actual issue was fixed by @ThisIsAreku b005f3c3
